bool is_sad
---
bool resp_is_sad
